services/upload/pinata.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+import axios from 'axios'
+
+const endpoint = 'https://api.pinata.cloud/pinning/pinFileToIPFS'
+
+export const uploadToPinata = async (
+  fileArray: File[],
+  pinataApiKey: string,
+  pinataSecretKey: string,
+  fileType?: string,
```

Comment:
We should enforce the file type in here always

```suggestion
type UploadFileType = 'metadata' | 'images'
fileType: UploadFIleType,
```